### PR TITLE
layout_calendar

### PR DESCRIPTION
**Pull request by iLinuxx**
https://ilinuxx.github.io/layout_calendar/

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great start! Let's fix some issues: Modifiers related to the weekdays don't work.

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./blocks/utils/variables";
+
+@import "blocks/calendar";
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:3]: The import statement should have a relative path: './blocks/calendar';

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    font-size: $font-size;
+    cursor: pointer;
+    transition: all $animation-duration;
+
+    &::before {
+      content: "";
+    }
+
+    &:hover {
+      background-color: green;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/calendar.scss]: Change the background-color from 'green' to 'pink' on hover according to the task description.

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    .calendar__day:nth-child(#{$count})::before {
+      content: "#{$count}";
+    }
+  }
+}
+
+@each $day, $start in $start-day-map {
+  .calendar--start-day--#{$day} {
+    .calendar__day:first-child {
+      margin-left: ($start - 1) * ($day-size+ 1px);
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/calendar.scss]: Use $calendar-gap instead of hardcoded '1px' for the margin calculation.

File: src/index.html [hunk 4]
```diff
@@ -4,9 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--30 calendar--start-day--tue"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html]:
```suggestion
    <div class="calendar calendar--month-length--30 calendar--start-day--tue">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more perfect

**Review by volodymyr-soltys97 — APPROVED** [review]
Nice work 👍

**Pull request closed without merging**